lib/active_model_serializers/test/serializer.rb

```diff
@@ -0,0 +1,122 @@
+module ActiveModelSerializers
+  module Test
+    module Serializer
+      extend ActiveSupport::Concern
+
+      included do
+        setup :setup_serialization_subscriptions
+      end
+
+      # Asserts that the request was rendered with the appropriate serializers.
+      #
+      #  # assert that the "PostSerializer" serializer was rendered
+      #  assert_serializer "PostSerializer"
+      #
+      #  # return a custom error message
+      #  assert_serializer "PostSerializer", "PostSerializer not rendered"
+      #
+      #  # assert that the instance of PostSerializer was rendered
+      #  assert_serializer PostSerializer
+      #
+      #  # assert that the "PostSerializer" serializer was rendered
+      #  assert_serializer :post_serializer
+      #
+      #  # assert that the rendered serializer starts with "Post"
+      #  assert_serializer %r{\APost.+\Z}
+      #
+      #  # assert that no serializer was rendered
+      #  assert_serializer nil
+      #
+      def assert_serializer(expectation, message = nil)
+        @assert_serializer.expectation = expectation
+        @assert_serializer.message = message
+        @assert_serializer.response = response
+        assert(@assert_serializer.matches?, @assert_serializer.message)
```

Comment:
Thanks for the review.

I don't think we need to care with RSpec and other libs at the moment, [Action Mailer uses Minitest methods](https://github.com/rails/rails/blob/68af99c942bf00f3a53df72555d2d5e3b94978cf/actionmailer/lib/action_mailer/test_helper.rb#L37) and [Active Job too](https://github.com/rails/rails/blob/68af99c942bf00f3a53df72555d2d5e3b94978cf/activejob/lib/active_job/test_helper.rb#L71). 

I think we can follow the same rationale, we are providing Minitest test helpers.